blocks/library/table/index.js

```diff
@@ -22,14 +23,14 @@ registerBlockType( 'core/table', {
 
 	attributes: {
 		content: {
-			type: 'array',
-			source: children( 'table' ),
-			default: [
-				<tbody key="1">
+			type: 'string',
+			source: html( 'table' ),
+			default: `
+				<tbody>
 					<tr><td><br /></td><td><br /></td></tr>
```

Comment:
Aside: Should these be marked as `data-mce-bogus` `br`? Seems like they should be ejected from the final markup.